File: Python/1046.py
(inicio, fin) = map(int, input().split(" "))
if(inicio == fin):
    print("O JOGO DUROU 24 HORA(S)")
elif(inicio > fin):
    print("O JOGO DUROU %d HORA(S)" % (24-inicio+fin))
elif(inicio < fin):
    print("O JOGO DUROU %d HORA(S)" % (fin-inicio))

File: Javascript/1046.js
var input = `2 16`;
let [inicio, fin] = input.split(/[\s]/).map((x) => Number(x));
if (inicio == fin) {
  console.log("O JOGO DUROU 24 HORA(S)");
} else if (inicio > fin) {
  console.log("O JOGO DUROU " + (24 - inicio + fin) + " HORA(S)");
} else if (inicio < fin) {
  console.log("O JOGO DUROU " + (fin - inicio) + " HORA(S)");
}
